### DISCUSSION TITLE
Adding components as a response to a slash command

### DISCUSSION THREAD

**Discussion by alfuwu** [Rapptz/discord.py#9363] [Q&A] [answered]
I've had a lot of trouble attempting to figure this out, and I still haven't. I would like to know if it's possible to add components (like some buttons) to a response to a slash command, as I would prefer using them over reactions.

Any help is appreciated :)

**Comment by lightlybakedman** [comment, marked as answer]
They are used the same way as when you normally send them. 
`await interaction.response.send_message(..., view = MyView())`

If you haven't worked with views before, see https://github.com/Rapptz/discord.py/tree/master/examples/views

**Reply by alfuwu** [reply]
thank you
